qpc/tests/scan/test_scan_add.py

```diff
@@ -113,11 +113,7 @@ def test_add_scan(self, caplog):
                 "id": 1,
                 "name": "scan1",
                 "sources": ["source1"],
-                "disable-optional-products": {
-                    "jboss-eap": False,
-                    "jboss-fuse": False,
-                    "jboss-brms": False,
```

Comment:
I'm so confused now. Why jboss-ws isn't part of the original payload 😱 
(just thinking out loud as this has nothing to do with your contribution)